cosmetics-web/app/forms/notification_wizard/delete_nano_material_form.rb

```diff
@@ -0,0 +1,25 @@
+module NotificationWizard
+  class DeleteNanoMaterialForm < Form
+    class NonEmptyArrayValidator < ActiveModel::EachValidator
+      def validate_each(record, attribute, value)
+        if value.blank? || value.reject(&:blank?).blank?
```

Comment:
Suggestion: Given that for `Array` both `#blank?` and `#empty?` are equivalent, I find using `#empty?` over the "Array" object , while keeping `#blank?` for the contained elements clearer to read in this context:
```suggestion
        if value.empty? || value.reject(&:blank?).empty?
```